easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'Bundle'
+
+name = 'scikit-image'
+version = '0.13.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
+description = """Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
+building upon numpy, scipy, and matplotlib. As a machine-learning module,
+it provides versatile tools for data mining and analysis in any field of science and engineering.
+It strives to be simple and efficient, accessible to everybody, and reusable in various contexts."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('Qhull', '2015.2'),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('Pillow', '5.0.0', versionsuffix),
+]
+
+exts_list = [
+    ('networkx', '2.1', {
+        'source_urls': ['https://github.com/networkx/networkx/archive/'],
```

Comment:
@RvDijk In existing easyconfigs we use `https://pypi.python.org/packages/source/n/networkx`, why the switch?

Reply:
@boegel at the time of installing I seemed to have received a wrong package, causing the installation to fail. However I can't reproduce this error anymore. I'll change back to the original source..